wordpress-mobile/WordPress-iOS
# Removes Aztec announcement VC from the App.

## Description

Fixes #8134 

Removes the Aztec announcement VC from the App, since Aztec is now live.

IMPORTANT: This is high priority, so we can make a new build for 8.8 ASAP.

## Testing:

1. Reset the device.
2. Launch the app.
3. Make sure the Aztec announcement popup is no longer shown.
